tripal/expression/__init__.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+from tripal.client import Client
+
+logging.getLogger("requests").setLevel(logging.CRITICAL)
+log = logging.getLogger()
+
+
+class ExpressionClient(Client):
+    """Manage Tripal expressions"""
+
+    def add_expression(self, organism, analysis, feature_match, file_type, file_path,
+                       biomaterial_provider=None, array_design=None, assay_id=None,
+                       acquisition_id=None, quantification_id=None, file_extension=None,
+                       start_regex=None, stop_regex=None, no_wait=False):
+        """
+        :type organism: str
+        :param organism: Organism Id
+
+        :type analysis: str
+        :param analysis: Id of the analysis
+
+        :type feature_match: str
+        :param organism: Match to features using either name or unique_name
+
+        :type file_type: str
+        :param file_type: Expresssion file type : column or matrix
+
+        :type file_path: str
+        :param file_path: Path to the expression file, or directory containing multiple expression files.
+
+        :type biomaterial_provider: str
+        :param biomaterial_provider: The contact who provided the biomaterial. (optional)
+
+        :type array_design: str
+        :param array_design: The array design associated with this analysis. This is not required if the experimental data was gathered from next generation sequencing methods. (optional)
+
+        :type assay_id: str
+        :param assay_id: The id of the assay associated with the experiment. (optional)
+
+        :type acquisition_id: str
+        :param acquisition_id: The id of the acquisition associated with the experiment (optional)
+
+        :type quantification_id: str
+        :param quantification_id: The id of the quantification associated with the experiment (optional)
+
+        :type file_extension: str
```

Comment:
Is this really needed? I don't see what it could be

Reply:
As it is, it's not really needed. However, we could add another cli to create these parameters (it allows the user to describe the experiment a little more, but they need to be defined beforehand)